content/blog/2020-05-05-updates-to-ragg-and-systemfonts/index.Rmarkdown

```diff
@@ -0,0 +1,169 @@
+---
+title: Updates to ragg and systemfonts
+description: |
+  We have recently updated some of our low-level graphics packages. Read on 
+  about what is new in ragg and systemfonts
+author: Thomas Lin Pedersen
+date: '2020-05-05'
+slug: updates-to-ragg-and-systemfonts
+categories:
+  - package
+tags:
+  - package
+  - ragg
+  - systemfonts
+authors:
+  - Thomas Lin Pedersen
+photo:
+  url: https://unsplash.com/photos/BVyNlchWqzs
+  author: Amador Loureiro
+---
+
+We're thrilled to announce the new releases of ragg and systemfonts. ragg is a package that provides a modern set of raster devices as alternatives to the ones you know from the grDevices package (e.g. `png()`). systemfonts is a low level package that provides font-related utilities to graphic devices, most importantly detecting and matching fonts to the correct font files across systems.
+
+While these are not the most flashy of releases, we thought it would be a good time to discuss some of our thoughts on the future of graphics in R, especially related to fonts.
+
+## ragg 0.2.0
+ragg is our current take on a modern set of raster devices for R. Together with the svglite package it provides alternatives to most of the devices that ships with base R. ragg is based on the AGG 2D graphics library, and provides both [high quality](https://ragg.r-lib.org/articles/ragg_quality.html) and [high performance](https://ragg.r-lib.org/articles/ragg_performance.html) output. One of its many virtues is that it provides high quality font support and automatically supports the different fonts installed on your system. This all happens through systemfonts as we'll discuss below.
+
+In this release we have cleared out a range of rendering bugs. The most user-visible change though, is the addition of a jpeg device (`agg_jpeg()`) for writing output to the jpeg file format. jpeg is usually not the best choice for the graphics one normally produce in R as it struggles to render line graphics without artifacts. For the standard type of plot, png (using `agg_png()`) will always be preferred. It does excel in compressing image data, so for this type of plot it can offer a very high compression with almost no noticeable degradation in quality:
```

Comment:
I think you should briefly explain the different between jpeg and png (i.e. jpeg is lossy), then say why that's usually not a good fit for R graphics, and then say when it is.